src/main/java/io/jenkins/plugins/coverage/CoverageProcessor.java

```diff
@@ -315,6 +312,57 @@ private HealthReport processThresholds(Map<CoverageReportAdapter, List<CoverageR
         return new HealthReport(score, localizeDescription);
     }
 
+
+    private void convertResultsToAction(CoverageResult coverageReport, HealthReport healthReport) throws IOException {
+        synchronized (CoverageProcessor.class) {
+            CoverageAction previousAction = run.getAction(CoverageAction.class);
+
+            if (previousAction != null) {
+                CoverageResult previousResult = previousAction.getResult();
+
+                int branchCount = previousAction.getBranchCount();
+                if (branchCount == 1) {
+                    String branchNameOfFirstCall = previousAction.getCacheBranchName();
```

Comment:
I'm not sure about the word `cache` here - is it really a cached? Maybe just `getBranchName`?

Reply:
This branch name is the name of the branch to be first called. So it actually saves the first branch name. But I agree with your, `cache` is not very good.

Reply:
Maybe I should change it to `branchNameOfFirstCall`?